package.json

```diff
@@ -28,7 +28,7 @@
     "redux": "3.7.2",
     "registry-image-widgets": "0.0.13",
     "requirejs": "2.1.22",
-    "term.js-cockpit": "0.0.8"
+    "term.js-cockpit": "0.0.9"
```

Comment:
Doesn't this need to be bumped to 0.0.10 too?

Reply:
Ah, I suppose not, as cockpit itself doesn't use the `showCursor()` call? But I wonder why this was blocked on the term.js 0.0.10 PR then.